src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -538,9 +544,18 @@ class Namespace:
     @staticmethod
     def init_namespace_dbs():
         db_conn = []
-        SonicDBConfig.load_sonic_global_db_config()
-        for namespace in SonicDBConfig.get_ns_list():
-            db = SonicV2Connector(use_unix_socket_path=True, namespace=namespace, decode_responses=True)
+        if not SonicDBConfig.isInit():
+            if multi_asic.is_multi_asic():
+                SonicDBConfig.load_sonic_global_db_config()
+            else:
+                SonicDBConfig.initialize()
```

Comment:
>initialize [](http://example.com/codeflow?start=30&length=10)

Use `load_sonic_db_config`, it will be mapped to `initialize` in sonic-swss-common.

Reply:
Modified to use load_sonic_db_config